CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 4.0.1
+
+- Adds mapping for ITEM_ALREADY_OWNED
+    https://github.com/RevenueCat/purchases-android/pull/220
```

Comment:
this is an addition in a public file, right? if that's the case, this should be 4.1.0

Reply:
It's just an added case to the switch that converts from Google Play Billing errors to our own errors, but we didn't create a new error. Should that still be a 4.1.0? I am not sure

Reply:
my bad, I didn't look closely and thought it was adding a public-facing error, but it's just privately mapping an extra error type to an existing error. 4.0.1 is fine.